plugin/completion/bin_complete.py

```diff
@@ -168,7 +168,7 @@ def init(self, view, settings):
         self.flags_dict[view.buffer_id()] = clang_flags
         log.debug(" clang flags are: %s", self.flags_dict[view.buffer_id()])
 
-    def complete(self, view, cursor_pos, show_errors):
+    def complete(self, view, cursor_pos, current_job_id):
```

Comment:
I don't think that's a good idea, as the actual implementations are quite different and I believe it's worth to have some additional info on these differences